mod_ci/controllers.py

```diff
@@ -826,11 +827,15 @@ def in_maintenance_mode(platform):
     platforms = TestPlatform.values()
     if platform not in platforms:
         return 'ERROR'
+    elif platform == 'linux':
```

Comment:
This is not really update-proof. There is a method in the enum class to convert from string to enum...